tests/codegen/compiletest-self-test/minicore-smoke-test.rs

```diff
@@ -0,0 +1,21 @@
+//! Basic smoke test for `minicore` test auxiliary.
+
+//@ use-minicore
+//@ revisions: meow
```

Comment:
When there's just a single revision, why use the revision system at all?

Reply:
Good point, this was there during testing, I forgot I only had one revision left.

Reply:
Removed as they were unnecessary.